app/views/articles/_v2_form.html.erb

```diff
@@ -17,11 +17,14 @@
   data-article="<%= article.to_json(only: %i[id title cached_tag_list published body_markdown main_image organization_id canonical_url updated_at], methods: %i[series all_series]) %>"
   data-organizations="<%= organizations&.to_json(only: %i[id name bg_color_hex text_color_hex], methods: [:profile_image_90]) %>"
   data-version="<%= version %>"
-  data-logo-svg='<%= logo_svg %>'>
+  data-logo-svg='<%= logo_svg %>'
```

Comment:
*(question)* Is this variable still being used somewhere or should it be removed?

Reply:
Nah, I forgot to remove it. Good catch!